Packages/SmithyTestUtil/Sources/RequestTestUtil/HttpRequestTestBase.swift

```diff
@@ -104,6 +104,13 @@ open class HttpRequestTestBase: XCTestCase {
         }
         return deconflictedHost
     }
+
+    public func urlPrefixFromHost(host: String) -> String {
+        guard !host.isEmpty, let hostCustomPath = URL(string: "http://\(host)")?.path else {
+            return ""
```

Comment:
why not just return nil here and check for nil in the middleware?

Reply:
completed!